solidity/contracts/libraries/operator/GroupSelection.sol

```diff
@@ -404,4 +418,21 @@ library GroupSelection {
             delete self.candidate[self.tickets[i]];
         }
     }
+
+    function reactiveTicketSubmissionRounds(
```

Comment:
We should add documentation explaining the magic we need to do inside.

Reply:
If we decide to hardcode the number of rounds as a constant, please ignore this comment.

Reply:
Not applicable now as we removed the dynamic submission timeout.